TFG-Labs/custom-shipping-step-by-items
# [WEB-451] Hide Suburb Banners

### What problem is this solving?
Given that these two fields are auto populated from the google, it needs to be removed from flow:

~1. Profile: Add Google Autofill Address~
2. Checkout: Add Google Autofill Address *

#### How it works

#### How to test it?
Navigate to checkout, add new address using Google search and the Banner should not show, for any other scenario should show
[Workspace](https://web451--thefoschini.myvtex.com/)

### Screenshots/Video example usage:
- Desktop
- Tablet
- Movil

https://github.com/user-attachments/assets/8f2f2fcf-ff30-42f0-919b-7df0e9d6946b


### Related to / Depends on
None
#### How does this PR make you feel? :link:
![]()
